metadata/thirdParty_axios.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Using axios dependency",
+  "description": "Replace HTTP requests sent through an HTTP Service client with calls to an HTTP library using the axios npm dependency",
```

Comment:
here is the attempt at the third party snippet title and description for Axios. Can you look over this to see if this is okay, feel free to suggest something else for the title and/or description.